pkg/resources/model/reconciler.go

```diff
@@ -203,6 +205,23 @@ func NewValidationReconciler(
 
 		resources.Logging.Status.Problems = nil
 
+		loggingsForTheSameRef := make([]string, 0)
+		for _, l := range resources.AllLoggings.Items {
+			if l.Name == resources.Logging.Name {
+				continue
+			}
+			if l.Spec.LoggingRef == resources.Logging.Spec.LoggingRef {
+				loggingsForTheSameRef = append(loggingsForTheSameRef, l.Name)
+			}
+		}
+
+		if len(loggingsForTheSameRef) > 0 {
+			problem := fmt.Sprintf("Deprecated behaviour! Other logging resources exist with the same loggingRef: %s",
```

Comment:
I think this message should announce what will happen on next major with these duplicated objects?

Reply:
If only I knew! :) Just kidding, ok will add that